packages/terra-signature/src/Signature.jsx

```diff
@@ -115,7 +115,11 @@ class Signature extends React.Component {
 
     this.canvasRect = this.canvas.getBoundingClientRect();
 
-    this.addLine(event.pageX - this.canvasRect.left, event.pageY - this.canvasRect.top, false);
+    if (event.type === 'touchstart') {
```

Comment:
are there any tests you could write for this?

Reply:
https://github.com/cerner/terra-core/pull/3436/files/61110e29625b4f0be9cdb948403e8abf0c183ce6#diff-101895006107b419f432f22c93db224b2ab5f26689e324d4a7e89f1914ed3604R104

This PR just contains one check that wasn't made during the last PR for this bug. Similar to all of these changes in mouseInBounds and mouseXY. We were not able to identify or write any of tests for this last time so I do not think there are any.